LRQA-77511

deborasoliveira opened the pull request:
https://issues.liferay.com/browse/LRQA-77511

crodriguezy committed:
LPS-142380 buildLang

crodriguezy committed:
LPS-142380 SV

crodriguezy committed:
LPS-142380 Add validation in ExpandoColumnLocalServiceImpl

crodriguezy committed:
LPS-142380 Add test

dantewang committed:
LPS-142380 Use proper method to add the attribute with default value to make sure validation is executed before creating the value

dantewang committed:
LPS-142380 Show error message on UI

dantewang committed:
LPS-142380 SF, inline

dantewang committed:
LPS-142380 SF, remove unnecessary try-rethrow

dantewang committed:
LPS-142380 Use ReflectionUtil to keep the exception instead of wrapping it in a RuntimeException; in this change's case, the UI won't be able to show the error message if the exception is rethrown as RuntimeException.

Commit:
LPS-150666 Add TheCloseButtonWorks test

Commit:
LPS-150666 Auto SF

Commit:
LPS-150666 Add property quarantine

Commit:
LPS-150666 SF

jgarcialr committed:
LPS-142380 SV

jgarcialr committed:
LPS-142380 rename exceptions

jgarcialr committed:
LPS-142380 Add validation in ExpandoValueLocalServiceImpl

dantewang committed:
LPS-161850 Directly use ConfigurationAdmin with the help of ConfigurationTestUtil like what's done in other tests, remove CreatingConfigurationHolder

dantewang committed:
LPS-161850 Directly use ServiceTracker, remove the holder

jgarcialr committed:
LPS-142380 remove calls to ValueDataException and use new static exceptions

dantewang committed:
LPS-161850 Directly use ConfigurationAdmin with the help of ConfigurationTestUtil like what's done in other tests, remove CreatingFactoryConfigurationHolder

dantewang committed:
LPS-161850 Inline ThreadContextClassLoaderCloseableHolder, there are a lot of usages of swapping the context class loader, but many of them can not be replaced by a try-with-autocloseable

dantewang committed:
LPS-161850 Inline AwaitingConfigurationHolder into PortalK8sAgentImplTest, prepare for next

dantewang committed:
LPS-161850 Fix test: the new method uses ConfigurationAdmin.listConfigurations which doesn't create a configuration when it doesn't exist, so we should assert null directly. Before this, the test uses "getConfiguration", which does create a new one when it doesn't exist.

rodrigopaulino committed:
LPS-162456 Add new column enableObjectEntryHistory to object definition

rodrigopaulino committed:
LPS-162456 Change places where it is used

rodrigopaulino committed:
LPS-162456 Auto SF

rodrigopaulino committed:
LPS-162456 Validate enableObjectEntryHistory

rodrigopaulino committed:
LPS-162456 Fix tests

rodrigopaulino committed:
LPS-162456 Add feature flag

rodrigopaulino committed:
LPS-162456 Include enableObjectEntryHistory to service methods

rodrigopaulino committed:
LPS-162456 Add new enableObjectEntryHistory column in upgrade process

Commit:
LPS-160278 User with no permissions can interact with the subscribe button

rodrigopaulino committed:
LPS-162459 Audit object entries

Commit:
LPS-159824 Add JSON file to test skippable on walkthrough

Commit:
LPS-159824 Add addWalkthroughPageAndEnablePopoverMode macro

Commit:
LPS-159824 Add CanEnableSkippable testcase

Commit:
LPS-159824 Refactor tests with json file with multiple pages

kresimir-coko committed:
LPS-162476 Update tests

kresimir-coko committed:
LPS-162476 Export misc utils

kyle-miho committed:
LRQA-76996 Add entity to test creating redundant unique indexes

kyle-miho committed:
LRQA-76996 buildService

kyle-miho committed:
LRQA-76996 Add testcase to cover buildService removing redundant unique indexes from indexes.sql

This reverts commit d34a8265024709fbe0ba34cc82c43e70b18cbfc7.

cgoncas committed:
LPS-160278 Add new method isDisabled to the ContentDashboardItemAction interface.

This method will disable the action if it is true.

cgoncas committed:
LPS-160278 Implement isDisabled method for Subscribe and UnSubscribe action.

cgoncas committed:
LPS-160278 Use the isDisabled method in the web module, instead implementing it directly in GetContentDashboardItemInfoMVCResourceCommand.

markocikos committed:
LPS-163597 Add new labels

markocikos committed:
LPS-163597 Implement UI to delete an FDS Custom View

markocikos committed:
LPS-163597 Implement REST endpoint to delete an FDS Custom View

seltong committed:
LPS-162276 Fetch the asset creator user

Commit:
LPS-159824 add setupWalkthroughPage macro

Commit:
LPS-159824 rename WalkthroughSiteLevel.testcase

Commit:
LPS-159824 Refactor testcases for new name of macro

kresimir-coko committed:
LPS-162476 Fix wrong import

Commit:
LPS-160278 Add double check in FE

Commit:
LPS-160278 Add test for disabled Susbcribe btn

eduardo-diniz committed:
LPS-163239 Creating Product Performance Product Filters

RafaelCRC committed:
LPS-162687 (account-service): Remove AccountEntry name shortening

ealonso committed:
LPS-163798 baseline

ealonso committed:
LPS-163798 Auto

ealonso committed:
LPS-163798 build.gradle

ealonso committed:
LPS-163798 Adapt LayoutUtilityPageEntry API, since plid should be masterLayoutPlid when we are adding and plid and type are not necessary when we are updating

ealonso committed:
LPS-163798 Make it consistent with LayoutPageTemplateEntryNameException

ealonso committed:
LPS-163798 Adds a new enum to represent the different utility pages types

ealonso committed:
LPS-163798 Adds new action command to add utility pages

ealonso committed:
LPS-163798 Adds new success message when we successfully create a utility page

ealonso committed:
LPS-163798 buildLang

ealonso committed:
LPS-163798 Adds creation menu to add new utility pages

ealonso committed:
LPS-163798 Adds the related layout when we are adding a new utility page

ealonso committed:
LPS-163798 Adds new UI to select a master layout for a utility page

murilostodolni committed:
LPS-163756 Add variables to system object payload

brianchandotcom committed:
LPS-142380 Sort

brianchandotcom committed:
LPS-142380 Like this

brianchandotcom committed:
LPS-142380 1.) Inner classes for exceptions don't end with Exception and 2.) match ValueDataException.MustInformDefaultLocale

brianchandotcom committed:
LPS-142380 Consistency

ealonso committed:
LPS-163798 Uses new exceptions

ealonso committed:
LPS-163798 Moves action to group resource actions, like ADD_LAYOUT, ADD_LAYOUT_BRANCH, ...

Commit:
LPS-159824 Refactoring to update the new name of json file

rodrigopaulino committed:
LPS-162459 baseline

michaelmdelcavalcanti committed:
LPS-141479 Created check to verify that the results are from updated versions of the workflow

murilostodolni committed:
LPS-163756 Fix return when object field name is different from DBColumnName

brookedalton committed:
LPS-162686 Add macro to change blueprint locale

brookedalton committed:
LPS-162686 Adjust macro to open localized page

brookedalton committed:
LPS-162686 Add testcase for Blueprint title/description language translation

oliv-yu committed:
LPS-164160 learn-resources: Add learn more link for tag and category facet

https://issues.liferay.com/browse/LPS-164160

patriciajeanperez committed:
LPS-162704 Remove search in Account macros

ealonso committed:
LPS-163961 Replace icon-vertical-card used for uploaded documents with clay vertical card

ealonso committed:
LPS-163961 Replace liferay-frontend:horizontal-card with clay:horizontal-card

Commit:
LPS-163401 Update functional test on cascade deletion type and sf

ricardocousolr committed:
LPS-159600 Add test

kresimir-coko committed:
LPS-162476 Properly export cookie types

sergiojimcos committed:
LPS-163790: Convert assetLibraryGroupId into assetLibraryId in Actions

ealonso committed:
LPS-163961 Adapt logic to take into account actions in vertical card, Copy logic from DDMStructrureElementsDefaultPropsTransformer.js

markocikos committed:
LPS-163597 Return 204 NO_CONTENT, standard for a DELETE endpoint

sergiojimcos committed:
LPS-163766: Update getAssetLibraryContentTemplatesPage action using asset-library

Commit:
LPS-159824 SF

lfbesada committed:
LPS-162398 Add an upgrade process in order to complete the urlTitle in those global article versions that have an empty urlTitle

lfbesada committed:
LPS-162398 Register the upgrade step

lfbesada committed:
LPS-162398 Update the required schema version

lfbesada committed:
LPS-162398 Use the latest article urlTitle as urlTitle if there is not urlTitle and is the company group or staging company group

lfbesada committed:
LPS-162398 Add asserts to the existing tests related to the friendlyURLs of the global articles to ensure that the article urlTitle is not null when the article is updated and the friendlyURLMap is empty

eduardo-diniz committed:
LPS-161423 BAR Compression functionality

Commit:
LPS-150666 Fix testcase CP2KeyDetailsPortal

Commit:
LPS-150666 Auto SF

MarinhoFeliphe committed:
LPS-164091 Add searh query parameter

MarinhoFeliphe committed:
LPS-164091 Build Rest

MarinhoFeliphe committed:
LPS-164091 Semver

Commit:
LPS-163784 Convert method references to field references

Commit:
LPS-163784 Fix tests

marianoalvarosaiz committed:
LPS-163977 Use PrefsPropsUtil to check USERS_REMINDER_QUERIES_ENABLED since they can be modified through instance settings.

marianoalvarosaiz committed:
LPS-163977 Use PrefsPropsUtil to check USERS_REMINDER_QUERIES_REQUIRED since they can be modified through instance settings.

Hanlf committed:
LPS-163961 Update xpath after LPS-163961

p2kmgcl committed:
LPS-164080 Avoid errors if fragment does not exist

jkappler committed:
LPS-164088 Copy Versions internal class to a new class called ContentDashboardItemVersion

jkappler committed:
LPS-164088 Replace usage

jkappler committed:
LPS-164088 Standardize namings

crodriguezy committed:
LPS-142380 Use specific types in inner exception classes and construct the message there

victorg1991 committed:
LPS-153155 Use fetch directly to avoid doing unneeded requests

MarinhoFeliphe committed:
LPS-164091 Cover searh by using integration test

p2kmgcl committed:
LPS-164109 Add fragmentEntryLinkNamespace

p2kmgcl committed:
LPS-164109 Replace var with const

jkappler committed:
LPS-164088 Sort

Commit:
LPS-150666 Fix CP2KeyDetailsPortal tests

p2kmgcl committed:
LPS-164118 Flatten only on first render

p2kmgcl committed:
LPS-164118 Memoize contentRequestBody

p2kmgcl committed:
LPS-164118 Flex is not needed here

p2kmgcl committed:
LPS-164118 Avoid centering the menu on smaller viewports

p2kmgcl committed:
LPS-164118 Do not reposition input to avoid unwanted scroll

p2kmgcl committed:
LPS-164118 Export useSessionState

rodrigopaulino committed:
LPS-162456 buildREST

rodrigopaulino committed:
LPS-162456 buildService

p2kmgcl committed:
LPS-164118 Store selectedMenuItemId in session

crodriguezy committed:
LPS-142380 SF

dsitu committed:
COMMERCE-9777 Remove orphaned notes.jsp

p2kmgcl committed:
LPS-164118 Store scroll in session

gislayne-vitorino committed:
LPS-163311 (portal-settings-web) Added attribute to aui:select tag

ealonso committed:
LPS-163798 Moves action to group resource actions, like ADD_LAYOUT, ADD_LAYOUT_BRANCH, ...

lfbesada committed:
LPS-162398 SF, sort based on the order in the portal-tables.sql

ytuik committed:
LPS-164147 Create the picklist for the Request Object

ytuik committed:
LPS-164147 Create the Request Object

luisacmn committed:
LPS-163141 Text fixes after QA review

lfbesada committed:
LPS-162398 SF, sort and declare as used

luisacmn committed:
LPS-163141 SF

lfbesada committed:
LPS-162398 SF, sort

oliv-yu committed:
LPS-163949 search-experiences-web: Sort parameters inside util function validateRequired

Commit:
LPS-164146 Adjust the message on the page of Knockout

ealonso committed:
LPS-164157 Supports custom cards on item selector view descriptor

Commit:
LPS-161652 Add keywords for download alert

murilostodolni committed:
LPS-163756 Fix toDTO method call

arthurchan35 committed:
LPS-162500 Replace DB lock, as it generates unneccesay hybernate log

JanisZhang committed:
LPS-164192 Disable auto import default references

JanisZhang committed:
LPS-164192 Auto generated

JanisZhang committed:
LPS-164192 Fix missing reference manually

arthurchan35 committed:
LPS-162500 Let background task to do all token refresh tasks:

1. last solution has a problem that can potentially lock many users when many users try to refersh at same time.
2. to avoid session extension task in session check thread

JanisZhang committed:
LPS-164193 Disable auto import default references

JanisZhang committed:
LPS-164193 Auto generated

JanisZhang committed:
LPS-164193 Fix missing reference manually

JanisZhang committed:
LPS-164196 Disable auto import default references

JanisZhang committed:
LPS-164196 Auto generated

JanisZhang committed:
LPS-164196 Fix missing reference manually

JanisZhang committed:
LPS-164199 Disable auto import default references

JanisZhang committed:
LPS-164199 Auto generated

JanisZhang committed:
LPS-164199 Fix missing reference manually

JanisZhang committed:
LPS-164200 Disable auto import default references

JanisZhang committed:
LPS-164200 Auto generated

JanisZhang committed:
LPS-164200 Fix missing reference manually

JanisZhang committed:
LPS-164201 Disable auto import default references

JanisZhang committed:
LPS-164201 Auto generated

JanisZhang committed:
LPS-164201 Fix missing reference manually

dantewang committed:
LPS-161850 Remove the timeout and move service registration to the beginning to be consistent with FileInstall*Test; without this change, the second count down will always time out, adding unnecessary blocking time to the test execution.

dantewang committed:
LPS-161850 Extract common logic into ConfigurationTestUtil

dantewang committed:
LPS-161850 SF, inline

dantewang committed:
LPS-161850 Sem Ver

dantewang committed:
LPS-161850 Deploy portal-k8s-agent-test-dependencies like *-test-util, and enable PortalK8sAgentImplTest

lfbesada committed:
LPS-163216 Prevent from reset the draft layout prototype if the export of draft layout is disabled

lfbesada committed:
LPS-163216 When an existing layout is imported and the export of the draft layouts is disabled, then we should discard the current draft by copying the imported layout

lfbesada committed:
LPS-163216 When publishing a draft layout, if the layout set prototype link is active, we should preserve the published layout "last merge" related type settings properties

ealonso committed:
LPS-164157 baseline

liuxingye committed:
LPS-164244 Use suitable way to assert disabled checked box

liuxingye committed:
LPS-164236 Update function for checking box when import

JoyceWang08211 committed:
LRQA-77543 Include staging tests in ExportImport module relevant suite

liuxingye committed:
LPS-164217 Update condition for assert enabled label

victorg1991 committed:
LPS-164258 Add labels to the menu display fieldsets

victorg1991 committed:
LPS-164258 buildLang

victorg1991 committed:
LPS-164258 Show label if the color picker is used in the configuration

kresimir-coko committed:
LPS-162476 Revert import in non-module portlets

lfbesada committed:
LPS-164241 Hide form fragments in display page templates

lfbesada committed:
LPS-164263 Add a new interface to check the info permission

lfbesada committed:
LPS-164263 Register the InfoPermissionProvider services

lfbesada committed:
LPS-164263 Export the new package

lfbesada committed:
LPS-164263 Baseline

markocikos committed:
LPS-163597 Namesmith

sandrodw3 committed:
LPS-164283 Add tooltip with pages url

arthurchan35 committed:
LPS-162250 avoid race condition across nodes

sandrodw3 committed:
LPS-164258 Rename

riccardo-ferrari committed:
LRAC-11869 analytics-settings-rest-impl Add client exception

riccardo-ferrari committed:
LRAC-11869 analytics-settings-rest-impl Data source connection management

shuyangzhou committed:
LPS-161850 Deploy test dependencies module the normal way to avoid ci deployment race condition fail.

sandrodw3 committed:
LPS-164241 Remove poshi test

ozysouza committed:
LPS-164241 Remove poshi test

victorg1991 committed:
LPS-164262 Pass the display object to calculate configuration values and context objects

victorg1991 committed:
LPS-164262 Adapt usages

riccardo-ferrari committed:
LRAC-11869 analytics-settings-rest-impl Settings Manager Set multi value keys default

welitonpe committed:
LPS-164310 Created TestrayIcons component and spritemap

welitonpe committed:
LPS-164311 Update Translations

welitonpe committed:
LPS-164311 update styles

welitonpe committed:
LPS-164311 Replace icons on header

welitonpe committed:
LPS-164311 Replace icons on sidebar

welitonpe committed:
LPS-164311 SF

evanilson-santana committed:
LPS-161894 - Add CanCreateObjectWithRelationshipParameter Test

evanilson-santana committed:
LPS-161895 - Add CanEditRelationshipParameter Test

evanilson-santana committed:
LPS-161896 - Add CannotSeeParameterWhenNotRequiredByRelationship Test

brianchandotcom committed:
LPS-163141 upload

brianchandotcom committed:
LPS-161423 upload

brianchandotcom committed:
LPS-163790 SF

brianchandotcom committed:
LPS-141479 SF

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

deborasoliveira commented:
ci:test:sf

brianchandotcom committed:
LPS-163798 SF

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/5123095f6f068154006c653d74395bd333de8ec2">5123095f6f068154006c653d74395bd333de8ec2</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/deborasoliveira/liferay-portal/tree/LRQA-77511">LRQA-77511</a><br/>Branch GIT ID: <a href="https://github.com/deborasoliveira/liferay-portal/commit/ad582cf23c8f224000a4885910afcab66e3b4df1">ad582cf23c8f224000a4885910afcab66e3b4df1</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/3353/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/3353/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/3353/">test-portal-source-format#3353</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/3353/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/AfonsoHSantos/liferay-portal/pull/154">AfonsoHSantos#154</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1992888153">[master] ci:test:sf - deborasoliveira &gt; AfonsoHSantos - PR#154 - 2022-09-28[12:28:54]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/12190/">publish-testray-report#12190</a><br/></div></div>

Commit:
LPS-164325 Create MyEVP Table Employee View

brianchandotcom committed:
LPS-162398 SF

MarinhoFeliphe committed:
LPS-162276 Only populate the asset creator and assigned user for notifications

brianchandotcom committed:
LPS-163766 Rename

brianchandotcom committed:
LPS-163766 Rename

brianchandotcom committed:
artifact:ignore object-web 1.0.113 prep next

brianchandotcom committed:
LRAC-11869 SF

brianchandotcom committed:
LPS-162459 SF

Commit:
LPS-158872 Fix CanViewDefaultResourcesOnSystemObjects test

brianchandotcom committed:
LPS-163798 My bad

ozysouza committed:
LPS-164269 Update locators

Commit:
LPS-158872 Fix CanViewEmptyValuesDefaultOnColumnsOnSystemObjects test

ViniLopes20 commented on portal-web/test/functional/com/liferay/portalweb/tests/MyDmTests/MyDMTests.testcase:
Oi @deborasoliveira, você precisa criar o DM usando a UI e não usando macro por API

ViniLopes20 commented on portal-web/test/functional/com/liferay/portalweb/tests/MyDmTests/MyDMTests.testcase:
Oi @deborasoliveira, mande somente o teste que você está fazendo no momento

Commit:
LPS-164040 Fix typo

brianchandotcom committed:
LPS-164091 SF

brianchandotcom committed:
LPS-164311 upload

Commit:
LPS-164355 Modify cases to adapt new behavior

lfbesada committed:
LPS-164087 Prevent from export an empty formLayout JSON

lfbesada committed:
LPS-164087 Rename variables to avoid ambiguity with fields of com.liferay.headless.delivery.dto.v1_0.Layout class.

Using the old names as the variable name we have compilation error as "incompatible types:  flexWrap  cannot be converted to String". That's because the homonyms field of Layout.flexWrap is shadowing the local variable flexWrap.

Commit:
LPS-160278 Auto SF

JoyceWang08211 committed:
LPS-164216 Close redundant window

ealonso committed:
LPS-163961 We should use custom getInputName

dantewang committed:
LPS-163784 SF, sort

lfbesada committed:
LPS-164129 Prevents NPE when fragmentEntryKey is null

victorg1991 committed:
LPS-164262 Baseline

Commit:
LPS-162683 Add test to assert the unrelated changes don't display

Commit:
LPS-164211 Remove MariaDB temporarily due to known issue

liferay-translation committed:
LPS-77699 Update Translations

brianwulbern committed:
LRQA-51488 Fix typos

brianwulbern committed:
LRQA-51488 Regen

brianchandotcom committed:
LPS-162500 Order seems to matter

brianchandotcom committed:
LPS-162500 Rename

brianchandotcom committed:
LPS-162500 auto SF

brianchandotcom committed:
LPS-162500 don't pass it in

deborasoliveira commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d28bde84afca8601cee4ff88ff593e158a9f3645">d28bde84afca8601cee4ff88ff593e158a9f3645</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/deborasoliveira/liferay-portal/tree/LRQA-77511">LRQA-77511</a><br/>Branch GIT ID: <a href="https://github.com/deborasoliveira/liferay-portal/commit/ecdd246ee4e025aebe9783731ca7fa33b004e57e">ecdd246ee4e025aebe9783731ca7fa33b004e57e</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/3365/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/3365/jenkins-report.html">here</a>.</h5><pre>     [exec] Generating types (19 of 25): item-selector-taglib
     [exec] Generating types (20 of 25): layout-admin-web
     [exec] Generating types (21 of 25): @liferay/notification-web
     [exec] Generating types (22 of 25): @liferay/oauth2-provider-web
     [exec] Generating types (23 of 25): @liferay/object-dynamic-data-mapping-form-field-type
     [exec] Generating types (24 of 25): @liferay/object-js-components-web
     [exec] Generating types (25 of 25): @liferay/object-web
     [exec] Preflight check failed:
     [exec] checkTypeScriptTypeArtifacts() failed: Error: Type generation was successful but 1 project was out of date. Please commit updated versions of the artifacts in the projects listed above.
     [exec] 
     [exec] 1 of 3 jobs failed
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] error Command failed with exit code 1.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2022-09-29 17:06:52.752.
     [exec] 3 actionable tasks: 2 executed, 1 up-to-date
     [exec] 
     [exec] See the profiling report at: file:///opt/dev/projects/github/liferay-portal/build/reports/profile/profile-2022-09-29-10-04-35.html
     [exec] A fine-grained performance profile is available: use the --scan option.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$3(ExecuteActionsTaskExecuter.java:186)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:268)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:184)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:173)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/3365/">test-portal-source-format#3365</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/3365/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/AfonsoHSantos/liferay-portal/pull/154">AfonsoHSantos#154</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1993969317">[master] ci:test:sf - deborasoliveira &gt; AfonsoHSantos - PR#154 - 2022-09-29[10:01:27]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/12232/">publish-testray-report#12232</a><br/></div></div>

ViniLopes20 commented on portal-web/test/functional/com/liferay/portalweb/tests/MyDmTests/MyDMTests.testcase:
Oi @deborasoliveira, eu acho que você poderia fazer essa deleção via API

ViniLopes20 commented on portal-web/test/functional/com/liferay/portalweb/tests/MyDmTests/MyDMTests.testcase:
Oi @deborasoliveira, eu acho que você poderia fazer essa deleção via API

deborasoliveira commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d01f3d2db206f22e57e437fbfa2e1ac7554ee677">d01f3d2db206f22e57e437fbfa2e1ac7554ee677</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/deborasoliveira/liferay-portal/tree/LRQA-77511">LRQA-77511</a><br/>Branch GIT ID: <a href="https://github.com/deborasoliveira/liferay-portal/commit/ecdd246ee4e025aebe9783731ca7fa33b004e57e">ecdd246ee4e025aebe9783731ca7fa33b004e57e</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-source-format/3223/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/3223/jenkins-report.html">here</a>.</h5><pre>     [exec] 
     [exec] Please commit them.
     [exec] 
     [exec] M modules/apps/oauth2-provider/oauth2-provider-web/types/src/main/resources/META-INF/resources/js/OAuth2Client.d.ts
     [exec]  M modules/apps/oauth2-provider/oauth2-provider-web/types/src/main/resources/META-INF/resources/js/liferay.d.ts
     [exec] 
     [exec] Generating types (23 of 25): @liferay/object-dynamic-data-mapping-form-field-type
     [exec] Generating types (24 of 25): @liferay/object-js-components-web
     [exec] Generating types (25 of 25): @liferay/object-web
     [exec] Preflight check failed:
     [exec] checkTypeScriptTypeArtifacts() failed: Error: Type generation was successful but 2 projects were out of date. Please commit updated versions of the artifacts in the projects listed above.
     [exec] 
     [exec] 1 of 3 jobs failed
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] error Command failed with exit code 1.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2022-09-29 19:57:43.188.
     [exec] 3 actionable tasks: 2 executed, 1 up-to-date
     [exec] 
     [exec] See the profiling report at: file:///opt/dev/projects/github/liferay-portal/build/reports/profile/profile-2022-09-29-12-55-29.html
     [exec] A fine-grained performance profile is available: use the --scan option.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$3(ExecuteActionsTaskExecuter.java:186)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:268)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:184)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:173)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/3223/">test-portal-source-format#3223</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/3223/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/AfonsoHSantos/liferay-portal/pull/154">AfonsoHSantos#154</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1994074695">[master] ci:test:sf - deborasoliveira &gt; AfonsoHSantos - PR#154 - 2022-09-29[12:52:21]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/12539/">publish-testray-report#12539</a><br/></div></div>

deborasoliveira committed:
LRQA-77511 Add CanCreateDM Testcase

deborasoliveira commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d01f3d2db206f22e57e437fbfa2e1ac7554ee677">d01f3d2db206f22e57e437fbfa2e1ac7554ee677</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/deborasoliveira/liferay-portal/tree/LRQA-77511">LRQA-77511</a><br/>Branch GIT ID: <a href="https://github.com/deborasoliveira/liferay-portal/commit/661f95d2aa71200e30261438cffff4bed7692129">661f95d2aa71200e30261438cffff4bed7692129</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-source-format/3669/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/3669/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/3669/">test-portal-source-format#3669</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/3669/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/AfonsoHSantos/liferay-portal/pull/154">AfonsoHSantos#154</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1994093738">[master] ci:test:sf - deborasoliveira &gt; AfonsoHSantos - PR#154 - 2022-09-29[13:36:05]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/publish-testray-report/13653/">publish-testray-report#13653</a><br/></div></div>

deborasoliveira commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d01f3d2db206f22e57e437fbfa2e1ac7554ee677">d01f3d2db206f22e57e437fbfa2e1ac7554ee677</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/deborasoliveira/liferay-portal/tree/LRQA-77511">LRQA-77511</a><br/>Branch GIT ID: <a href="https://github.com/deborasoliveira/liferay-portal/commit/7652a0889903b6ab4bbd6c26dad95a2623204997">7652a0889903b6ab4bbd6c26dad95a2623204997</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-source-format/1921/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/1921/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-source-format/1921/">test-portal-source-format#1921</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/1921/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/AfonsoHSantos/liferay-portal/pull/154">AfonsoHSantos#154</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1994134611">[master] ci:test:sf - deborasoliveira &gt; AfonsoHSantos - PR#154 - 2022-09-29[14:02:32]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/7205/">publish-testray-report#7205</a><br/></div></div>

ViniLopes20 reviewed (approved):
Muito bom @deborasoliveira, obrigado por isso 😄

AfonsoHSantos commented:
Muito bom @deborasoliveira  , parabéns!
